examples/arkode/F2003_parallel/CMakeLists.txt

```diff
@@ -17,13 +17,16 @@
 
 # Example lists are tuples "name\;nodes\;tasks\;type" where the
 # type is develop for examples excluded from 'make test' in releases
-set(FARKODE_examples
-    "ark_brusselator1D_task_local_nls_f2003\;--monitor\;1\;4\;develop\;2"
-    "ark_brusselator1D_task_local_nls_f2003\;--monitor --global-nls\;1\;4\;develop\;2"
-    "ark_brusselator1D_task_local_nls_f2003\;--monitor --explicit --tf 3\;1\;4\;develop\;2"
-    "ark_diag_kry_bbd_f2003\;\;1\;4\;develop\;2"
-    "ark_diag_non_f2003\;\;1\;4\;develop\;2"
-    "ark_heat2D_f2003\;\;1\;4\;develop\;2")
+
+if(SUNDIALS_INDEX_SIZE MATCHES "64")
+  set(FARKODE_examples
+      "ark_brusselator1D_task_local_nls_f2003\;--monitor\;1\;4\;develop\;2"
+      "ark_brusselator1D_task_local_nls_f2003\;--monitor --global-nls\;1\;4\;develop\;2"
```

Comment:
Looks like this example was updated to work with 32-bit indices should they be outside the "if 64"?